databases/week3/Craete_Databse.sql

```diff
@@ -0,0 +1,34 @@
+-- Active: 1660903523654@@127.0.0.1@3306@database_week3
+USE database_week3;
+SET NAMES UTF8MB4;
+CREATE TABLE `Meal` (
+  `id` INT(10) UNSIGNED NOT NULL AUTO_INCREMENT,
+  `title` VARCHAR(255) NOT NULL,
+  `description` TEXT NOT NULL,
+  `location` VARCHAR(255) NOT NULL,
+  `when` DATETIME,
+  `max_reservations` INT NOT NULL,
+  `price` DECIMAL NOT NULL,
+  `created_date` DATE,
+  PRIMARY KEY (`id`)
+) ENGINE=INNODB DEFAULT CHARSET=UTF8MB4 COLLATE=UTF8MB4_UNICODE_CI;
+
+CREATE TABLE `Reservation` (
+    `id` INT(10) UNSIGNED NOT NULL AUTO_INCREMENT PRIMARY KEY,
+    `number_of_guests` INT(10) NOT NULL,
+    `meal_id` INT(10) UNSIGNED NOT NULL,
+    `created_date` DATE,
+    `contact_phonenumber` VARCHAR(255) NULL,
+    `contact_name` VARCHAR(255) NOT NULL,
+    `contact_email` VARCHAR(255) NOT NULL,
+    CONSTRAINT `fk_reservation_meal` FOREIGN KEY (`meal_id`) REFERENCES `Meal` (`id`) ON DELETE CASCADE ON UPDATE CASCADE
+)ENGINE = InnoDB DEFAULT CHARSET = utf8mb4 COLLATE = utf8mb4_unicode_ci;
+CREATE TABLE `Review` (
+    `id` INT(10) UNSIGNED NOT NULL AUTO_INCREMENT PRIMARY KEY,
+    `title` VARCHAR(255) NOT NULL,
+    `description` TEXT NULL DEFAULT NULL,
+    `meal_id` INT(10) UNSIGNED NOT NULL,
+    `stars` INT(10) UNSIGNED NOT NULL,
+    `created_date` DATE ,
+    CONSTRAINT `fk_review_meal` FOREIGN KEY (`meal_id`) REFERENCES `Meal` (`id`) ON DELETE CASCADE ON UPDATE CASCADE
+)ENGINE = InnoDB DEFAULT CHARSET = utf8mb4 COLLATE = utf8mb4_unicode_ci;
```

Comment:
Good job here! We use parenthesis here when you create the name, but not when quering.

Reply:
ahmm you are right, I was confuse some of the sql file I have seen backtit `` in table name but some of the I couldn't.